adapter/noop2/noop.go

```diff
@@ -82,37 +84,41 @@ func (builder) ConfigureQuotaHandler(map[string]*quota.Type) error {
 
 ////////////////// Runtime Methods //////////////////////////
 
-var cacheInfo = adapter.CacheabilityInfo{
+var checkResult = adapter.CheckResult{
+	Status:        rpc.Status{Code: int32(rpc.OK)},
```

Comment:
So I was under the impression, adapters only interact with pkg/adapter. Is it okay to reference pkg/status here ?